src/elements/ngm-share-link.js

```diff
@@ -0,0 +1,110 @@
+import {LitElementI18n} from '../i18n';
+import {html} from 'lit-element';
+import {showWarning} from '../message';
+import i18next from 'i18next';
+import $ from '../jquery';
+import 'fomantic-ui-css/components/popup.js';
+import './ngm-i18n-content.js';
+import {SHORTLINK_HOST_BY_PAGE_HOST} from '../constants';
+
+class NgmShareLink extends LitElementI18n {
+
+  static get properties() {
+    return {
+      shortlink: {type: String}
```

Comment:
This is an internal property, so you should be able to use (not tested):
```suggestion
      shortlink: {attribute: false}
```